Realm/Realm/Handles/SessionHandle.cs

```diff
@@ -233,7 +234,7 @@ private static void HandleSessionError(IntPtr sessionHandlePtr, ErrorCode errorC
             }
             catch (Exception ex)
             {
-                Console.WriteLine(ex.Message);
+                Logger.Default.Log(LogLevel.Warn, $"An error has occurred while handling a session error: {ex}");
```

Comment:
This is a drive-by - it should have used the logger rather than Console.WriteLine-ing.